lib/index.js

```diff
@@ -4,6 +4,16 @@ var util = require('util');
 var EventEmitter = require('events').EventEmitter;
 
 /**
+ * routine errors
+ */
+var er = {
+  op              : 'nanomsg operation failed: ',
+  param           : 'parameter type err: ',
+  chan            : 'setting channel subscription string: ',
+  rmchan          : 'removing channel subscription string: ',
+}
```

Comment:
We'll let the formatter figure it out, when it lands.  I don't have my heart set on 80 char line limits.